.github/workflows/publish-latest.yml

```diff
@@ -1,4 +1,8 @@
-name: Publish to latest
+# Description: this workflow publishes UI packages to `latest`.
+#
```

Comment:
Do you think it would be helpful if we mention that once this workflow is done successfully, we'll see the "Last Publish" time changes here https://www.npmjs.com/package/@aws-amplify/ui ?

Reply:
Yes! I'll add that

Reply:
[c661373](https://github.com/aws-amplify/amplify-ui/pull/2337/commits/c6613733afdadf4b8ef2777dda9270cd880c91f4)